client/layout/index.jsx

```diff
@@ -395,6 +404,9 @@ export default withCurrentRoute(
 					sidebarIsHidden,
 					sectionName,
 			  } );
+		const needsColorScheme =
```

Comment:
do we need to check whether the sidebar is hidden? For example, I think DomainAndPlanPackageFlow doesn't need the color scheme 🤔

Reply:
Good idea, updated!